nwbwidgets/controllers/multicontroller.py

```diff
@@ -0,0 +1,32 @@
+from typing import Tuple, Dict
+
+import ipywidgets as widgets
+
+
+class MultiController(widgets.VBox):
```

Comment:
Can you make this inherit from `widgets.Box`? See 

https://ipywidgets.readthedocs.io/en/7.x/examples/Widget%20Styling.html#The-VBox-and-HBox-helpers

Reply:
Yes I started running into this on another PR as well